### DISCUSSION TITLE
Nomenclature and selection of Color Scheme for both Light & Dark mode

### DISCUSSION THREAD

**Discussion by subhranshudas** [shadcn-ui/ui#893] [Q&A]
Hello All,
Thanks for the awesome tooling!

**I am a design noob.** 

Can people explain the nomenclature of the colors (muted, accent, primary, secondary etc) and how do you go about generating a color scheme which works in both LIGHT & DARK mode?? How did you come up with the Shadcn/ui palette.

I did read a bit about it in https://www.radix-ui.com/colors
One thing i found very intuitive was how they just used the same name-number pattern for both light and dark mode. For e.g. 
text-gray-8 works for both light & dark, no need to understand the actual HSL colors. In that way i can simply assign the "vars" instead of the HSL colors which is not immediately understandable.

Please guide as to what is the Nomenclature system you are following and how to make dark mode work out of the box.

Thanks.

**Comment by cxa** [comment]
Hi, you can try this <https://realazy.com/enough-shadcn-ui-colors> built by me. 

I use `OKHSL` to generate colors for both light and dark modes. You can check the source code for some inspiration if you're interested.
